packages/components/src/components/popover/popover.scss

```diff
@@ -65,6 +65,13 @@
         }
     }
 }
+.accu-info-display__popover,
+.mobile-widget__item-popover {
+    max-width: 28rem;
+    @include mobile {
+        max-width: 30rem;
```

Comment:
Just curious, why is this bigger on mobile screen compare to desktop?

Reply:
As per design 
<img width="935" alt="Screenshot 2023-08-18 at 11 47 48 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/a9f007ee-b8c0-4aa2-baf8-9672b4b4a818">

Reply:
@kate-deriv to me, it looks like 33rem for mobile is closer to design:
<img width="1562" alt="Screenshot 2023-08-18 at 1 29 04 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/dbbeb1b7-3478-4da9-b79f-a8850ec47e5a">
<img width="1562" alt="Screenshot 2023-08-18 at 1 27 42 PM" src="https://github.com/binary-com/deriv-app/assets/103177211/c7de25a8-4b05-42ad-a028-c33ab78cb535">
please check :)

Reply:
@kate-deriv also according to design this bubble ends where the text ends but in your test link it goes outside of the trade params on the right side. please check this too🙏🏻